tests/dragonfly/cluster_test.py

```diff
@@ -88,6 +145,14 @@ async def wait_for_status(admin_client, node_id, status):
             await asyncio.sleep(0.05)
 
 
+async def check_for_no_state_status(admin_clients):
```

Comment:
I would make this function more generic
check_for_state_status(admin_clients, expected_status)